src/views/StandaloneOfficeAddressFiling.vue

```diff
@@ -75,13 +75,11 @@
             <header>
               <h1 id="address-change-header">Address Change</h1>
 
-              <p>
-                <span v-if="isCoop">Please change your Registered Office Address.</span>
-                <span v-if="isBComp">Please change your Registered Office Address and Records Address.</span>
-              </p>
+              <p v-if="isCoop">Please change your Registered Office Address.</p>
+              <p v-else-if="isBenBcCccUlc">Please change your Registered Office Address and Records Address.</p>
```

Comment:
No change to UI. Still looks like this:

![image](https://user-images.githubusercontent.com/10002889/202822346-0cca802a-3004-45bb-937b-dfc367cabf11.png)

Or this:

![image](https://user-images.githubusercontent.com/10002889/202822248-59669537-7330-4dc3-af59-9b09ae07b303.png)